pip/commands/show.py

```diff
@@ -12,6 +13,13 @@
 logger = logging.getLogger(__name__)
 
 
+_normalize_re = re.compile(r'[^a-z0-9-.]', re.I)
+
+
+def normalize_name(name):
+    return _normalize_re.sub('-', name.lower())
+
+
```

Comment:
This is already defined in https://github.com/pypa/pip/blob/develop/pip/utils/__init__.py#L189-L193